test/easylistTestCases.js

```diff
@@ -0,0 +1,60 @@
+const easylistTestCases = [
+    // blocking by address parts
+    // /banner/*/img
+    {'url': 'http://example/banner/foo/img', 'block': true, 'options': {}},
+    {'url': 'http://example/banner/foo/bar/img?param', 'block': true, 'options': {}},
+    {'url': 'http://example/banner//img/foo', 'block': true, 'options': {}},
+    {'url': 'http://example/banner/img', 'block': false, 'options': {}},
+    {'url': 'http://example/banner/foo/imgraph', 'block': false, 'options': {}},
+    {'url': 'http://example/banner/foo/img.gif', 'block': false, 'options': {}},
```

Comment:
Just curious here: what happens if you test url: `http://example/banner/*/img`

Reply:
This will match but only because of a separate bug. Wild card filters like that should not go into the bloom filter. It's on my list of bugs already.